src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
 // write your code here
+const numbers = document.getElementsByClassName('population');
```

Comment:
Ensure that elements with the class 'population' exist in the DOM to avoid runtime errors when accessing their text content.